addons/breathing/functions/fnc_treatmentAdvanced_pulseoximeterLocal.sqf

```diff
@@ -41,9 +42,22 @@ _patient setVariable [QGVAR(pulseoximeter), true, true];
         _SpO2 = 0;
     };
 
-
     [_patient, "quick_view", LSTRING(pulseoxi_Log)] call EFUNC(circulation,removeLog);
     [_patient, "quick_view", LSTRING(pulseoxi_Log), [round _HR, round _SpO2]] call ACEFUNC(medical_treatment,addToLog);
 }, 1, [_patient, _bodyPart]] call CBA_fnc_addPerFrameHandler;
 
 [_patient, "activity", LSTRING(pulseoxi_Log_2), [[_medic] call ACEFUNC(common,getName)]] call ACEFUNC(medical_treatment,addToLog);
+
+[_patient, _medic] spawn {
+    params ["_patient", "_medic"];
+    while {_patient getVariable [QGVAR(pulseoximeter), false]} do {
+        if(_patient getVariable [QGVAR(PulseOximeter_VolumePatient), false]) then {
+            private _hr = _patient getVariable [QACEGVAR(medical,heartRate), 80];
+            private _spO2 = _patient getVariable [QEGVAR(breathing,airwayStatus), 100];
+            if(_spO2 < GVAR(PulseOximeter_SpO2Warning) && _hr != 0) then {
+                playSound3D [QPATHTOF_SOUND(audio\pulseoximeter_warning.wav), _patient, false, getPosASL _patient, 4, 1, 15];
+                sleep 3;
```

Comment:
https://ace3.acemod.org/wiki/development/arma-3-scheduler-and-our-practices.html
in general sleep is banned, use correct cba fnc for that
also i think it can be included in pfh above (just change 1s interval to 3s) bc that sound is basically beeping when its "working" so it should be better that way 🤔 
what do you think @YetheSamartaka or @AtrixZockt

Reply:
Hard to tell for me cause i can't really see the code correct on my phone but it sounds good